organization/_posts/2018-01-11-startup.md

```diff
@@ -0,0 +1,170 @@
+        Software for Big Science journal](https://www.springer.com/physics/particle+and+nuclear+physics/journal/41781)
+        about publishing it there.
+        -   Pere wondered if this was really suitable for a research
+            journal. As we were approached by the editors we think
+            so - more of a review article.
+        -   Those who have known about this (Ghost Writers) are positive
+            about this.
+        -   This would provide a natural cut off for signing.
+        -   General conclusion is that we should pursue this with the
+            journal editors.
+
+-   Publication strategy for Individual WG Papers
+    -   We noted that conversion from Google Doc to LaTeX was far less
+        painful than had been thought so our recommendation for arXiv
+        publication for individual papers is updated. This was
+        followed to get the Software Development paper to arXiv late
+        last year.
+        -   [Advice on WG publication](/cwp/cwp-wg-papers.html).
+    -   We have to decide what to do when the WG paper is in a much less
+        developed state than the roadmap (Generators, ML) or only only
+        really exists in the roadmap (Security).
+    -   ### Software Development
+        -   Done as exemplar:
+            [https://arxiv.org/abs/1712.07959](https://arxiv.org/abs/1712.07959)
+    -   ### Simulation
+        -   Need to revisit the detailed paper ensure consistency. Then
+            will follow the procedure.
+    -   ### Machine Learning
+        -   To follow up.
+    -   ### Software Trigger and Event Reconstruction
+        -   Neither Vava nor David can attend today. Apologies. We plan
+            to pick up improvements to our text made during the final
+            editing of the cwp and hope to submit to the archive next
+            week. We already have completed one pass of gathering
+            authors and will do another.
+    -   ### Data Organisation, Management and Access
+        -   To follow up.
+    -   ### Data and Software Preservation
+        -   To follow up.
+    -   ### Data Analysis and Interpretation
+        -   To follow up.
+    -   ### Visualization
+        -   Ric (unable to attend today) will do final paper edits next
+            week, ready for last circulation before arXiv.
+    -   ### Event/Data Processing Frameworks
+        -   Liz at CERN next week, will follow up with Benedikt.
+    -   ### Careers, Staffing and Training
+        -   Dario will backport material from the roadmap.
+    -   ### Facilities and Distributed Computing
+        -   Clean up needed of the WG document.
+    -   ### Conditions Access
+        -   To follow up.
+    -   ### Generators
+        -   Needs some work from roadmap document. Should get CMS,
+            Durham IPPP inputs as well.
+    -   ### Security
+        - To follow up.
+-   John Harvey - what's the plan for producing a document for the LHCC?
```

Comment:
Triviality, sorry . I think you need a "<br>" at the end of this line if you want (better) the comment from Ian to come on a new line. I had a similar issue with .md yesterday.